app/models/weather.rb

```diff
@@ -0,0 +1,15 @@
+class Weather
+
+  attr_reader :summary, :temp, :wind
+  
+  def initialize(weather_atts={})
+    @summary = weather_atts[:weather]
+    @temp = weather_atts[:temperature_string]
+    @wind = weather_atts[:wind_mph]
+  end
+
+  def self.weather_info(lat,lon)
+    info = WeatherService.new.weather_info(lat,lon)[:current_observation]
+      Weather.new(info)
+  end
+end
```

Comment:
@Mcents This looks very nice, good catch on having `weather_atts={}` so that we don't error out if the API request returns nil.